### PR TITLE
CIV-10455 test

### PR DESCRIPTION
**Pull request by hmcts-version1-ignacio**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -10,7 +10,7 @@ def component = "citizen-ui"
 def camundaBranch = "master"
 def dmnBranch = "master"
 def waStandaloneBranch = "master"
-def ccdBranch = "master"
+def ccdBranch = "feat/CIV-10455"
```
Comment (laarmada) [inline Jenkinsfile_CNP:13]: thats correct?

File: src/main/routes/features/judgmentOnline/confirmYouHaveBeenPaidConfirmationController.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+import {constructResponseUrlWithIdParams} from 'common/utils/urlFormatter';
+import {AppRequest} from 'common/models/AppRequest';
+import {ClaimSummarySection} from 'form/models/claimSummarySection';
+import {PageSectionBuilder} from 'common/utils/pageSectionBuilder';
+
+const confirmationPageViewPath = 'features/judgmentOnline/judgment-online-confirmation-page.njk';
+
+const confirmYouHaveBeenPaidConfirmController = Router();
+export const getContent = (claimId: string): ClaimSummarySection[] => {
```
Comment (laarmada) [inline src/main/routes/features/judgmentOnline/confirmYouHaveBeenPaidConfirmationController.ts:14]: that should be in other file not related with a controller class

File: src/main/views/features/judgmentOnline/confirm-you-have-been-paid.njk [hunk 3]
```diff
@@ -0,0 +1,91 @@
+{% extends "claim-details-tpl.njk" %}
```
Comment (laarmada) [inline src/main/views/features/judgmentOnline/confirm-you-have-been-paid.njk:1]: are you using the new dashboard template?  claim-details-tpl-dashboard

**Review by laarmada — CHANGES_REQUESTED** [review]

File: Jenkinsfile_CNP (hunk 1 above)
Comment (hmcts-version1-ignacio) [reply Jenkinsfile_CNP:13]: Yes, this is the testing PR. Not going to merge this one.

File: src/main/routes/features/judgmentOnline/confirmYouHaveBeenPaidConfirmationController.ts (hunk 2 above)
Comment (hmcts-version1-ignacio) [reply src/main/routes/features/judgmentOnline/confirmYouHaveBeenPaidConfirmationController.ts:14]: Ok, I'll move it.

File: src/main/common/models/claim.ts [hunk 4]
```diff
@@ -170,6 +171,7 @@ export class Claim {
   orderDocumentId?: string;
   claimantEvidence: ClaimantEvidence;
   defendantResponseDocuments: SystemGeneratedCaseDocuments[];
+  joJudgmentPaidInFull?: CuiJudgmentPaidInFull;
```
[the remark this answers is not in the document]
Comment (hmcts-version1-ignacio) [reply src/main/common/models/claim.ts:174]: Ok

**Pull request closed without merging**